immix/src/llvm_stackmap/mod.rs

```diff
@@ -47,46 +36,16 @@ fn get_first_function_addr(mapptr: *const u8) -> *const u8 {
 
 #[derive(Debug, Clone)]
 pub struct Function {
-    pub addr: *const u8,
-    pub root_size: i32,
+    // pub addr: *const u8,
+    // pub offset:u32,
+    // pub root_size: i32,
```

Comment:
ditto

Reply:
Although those code were commented for performance reason, leave them here could help debugging if there's bug in stackmap reader.